docs/UserGuide.md

```diff
@@ -434,46 +434,91 @@ Examples:
 
 #### 7.4.1. Creating a Tag: `tag create`
 
-Creates a new tag
+You can create a new `TAG` and add it into the tag list.
 
 Format: `tag create t/TAG`
 
-Example:
-* `tag create t/family` creates a `family` tag.
+**Example Input in Command Box**
+```
+tag create t/friends
+```
+
+**Output in Command Result Box**
+```
+New Tag created: [friends]
+```
+
+   (insert image of a successful creation of the `friends` tag)
+
+Great! You have successfully learnt how to add your first `TAG` you have made. Now, you can start utilising the other tag features.
 
 <br>
 
 #### 7.4.2. Deleting a Tag: `tag delete`
 
-Deletes a tag.
+You can delete a `TAG` from the tag list.
 
 Format: `tag delete t/TAG`
 
 <div markdown="block" class="alert alert-info">
 **:information_source: Note:** When `TAG` is deleted, `TAG` is removed from all the contacts which previously had it.
 </div>
 
-Example:
-* `tag delete t/family` deletes the `family` tag.
+**Expected Output in Command Box**
+```
+tag delete t/friends
+```
+
+**Output in Command Result Box**
+```
+Tag deleted: [friends]
+```
+
+   (insert successful deletion of `Test2` tag)
+
+Wonderful! You have successfully deleted a tag.
 
 <br>
 
 #### 7.4.3. Editing a Tag: `tag edit`
 
-Renames an existing tag.
+If you make a mistake or want to update your `TAG`, you can simply update it with this command.
 
-Format: `tag edit t/TAG1 t/ TAG2`
+Format: `tag edit t/TAG1 t/TAG2`
 
-* `TAG1` is the current name of the tag and `TAG2` is the new name of the tag.
+<div markdown="block" class="alert alert-info">
+**:information_source: Note:** 
 
-Example:
-* `tag edit t/friend t/bestFriend` changes the friend tag to a bestFriend tag.
+* The new Tag must not have the same name as any other existing tags.
+* `TAG1` represents the current name of the tag and `TAG2` represents the new name of the tag.
+
```

Comment:
Might be good to let the users know that by editing the tag will change all the contacts and todos with `TAG1` to `TAG2`.

I am worried that they will think that this only affects one tag in one contact or something.

Reply:
I understand your concern. I'll make the changes.